pyproject.toml

```diff
@@ -9,7 +9,7 @@ readme = "readme.md"
 python = "^3.7.10"
 fire = "^0.4.0"
 pytest-cov = "^3.0.0"
-boto3 = "1.18.65"
+boto3 = "1.26.74"
 aiohttp = "^3.8.1"
 nest-asyncio = "^1.5.5"
 toml = "^0.10.2"
```

Comment:
You have to update `baram`'s version also. It is located above these.

Reply:
thanks for pointing out. I updated baram to 0.2.7 in last commit.